app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/WebRtcManager.kt

```diff
@@ -27,8 +30,11 @@ class WebRtcManager constructor(
         cameraEnumerator.deviceNames.asSequence()
             // prefer back-facing cameras
             .sortedBy { deviceName ->
-                if (cameraEnumerator.isBackFacing(deviceName)) 1
-                else 2
+                if (cameraEnumerator.isBackFacing(deviceName)) {
+                    1
+                } else {
+                    2
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/f1e62ee67a21a21873bbc26c24839e1cf2930e4c/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/WebRtcManager.kt#L36"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>